README.md

```diff
@@ -101,6 +101,11 @@ will be replaced with the newly built image if `os_images_upload` is set to `Tru
 
 `os_images_dib_venv`: Path to virtualenv in which to install DIB to build images.
 
+`os_images_cleanup`: __CAUTION__ (optional) boolean flag indicating whether or not the images should be deleted.
+Will take the list of images specified in `os_images_delete_list`. Defaults to `False`.
+
+`os_images_delete_list`: __CAUTION__ (optional) a list of images that will be deleted from Glance storage
```

Comment:
Do we need two variables here? I figure if os_image_delete_list is empty, we just don't do any deleting of things? I figure that is simpler, but I might be missing something here.

Reply:
This is a precuation, I don't want it to work with just providing the list.